test/fixtures/app/controllers/comments.js

```diff
@@ -1,30 +1,30 @@
 
 export default {
 
-  * index(next) {
+  * index(ctx, next) {
     return 'comments#index'
   },
 
-  * create(next) {
+  * create(ctx, next) {
     let name = 'trek'
     let age = 233
     return { name, age }
   },
 
-  * ['new'](next) {
-    this.body = 'comments#new'
+  * ['new'](ctx, next) {
+    ctx.body = 'comments#new'
   },
 
-  * show(next) {
-    this.body = 'comments#show'
+  * show(ctx, next) {
+    ctx.body = 'comments#show'
   },
 
-  * update(next) {
-    this.body = 'comments#update'
+  * update(ctx, next) {
```

Comment:
'generator functions' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'next' is defined but never used.